annotator/templates/video.html

```diff
@@ -51,7 +53,7 @@
                 {% if assignment_id != None %}
                 <form method="post" id="turk-form" action="https://{{ MTURK_SANDBOX|yesno:'workersandbox,www'}}.mturk.com/mturk/externalSubmit" style="margin-left: 300px">
```

Comment:
What does this line of code do for mturk? Is this also where iframes are supposed to be used?